tests/commands/new/test_validate_app_name.py

```diff
@@ -20,11 +19,16 @@ def test_valid_app_name(new_command, name):
 @pytest.mark.parametrize(
     'name',
     [
-        'hello world',  # space
-        'helloworld!',  # punctuation
-        '_helloworld',  # leading underscore
-        '-helloworld',  # leading hyphen
-        'existing',  # pre-existing directory
+        'hello world',
+        'helloworld!',
+        '_helloworld',
+        '-helloworld',
+        'existing',
+        'switch',
+        'pass',
+        'false',
+        'False',
+        'YIELD',
```

Comment:
The trailing comments are useful here because they describe what specific property is being tested. We should restore the old comments, and add new comment for the new test cases.